sci-mathematics/amath/amath-1.8.5.recipe

```diff
@@ -44,5 +44,5 @@ INSTALL()
 
 TEST()
 {
-	make test
+	make --test
```

Comment:
There's no `--test` switch for make.

Reply:
The switch is misplaced. It was intended as a switch for the application itself and not for the make file.
I'm am not so familiar with Github pull request. Should I make a new pull request in order to remove changes in line 47?

Reply:
Thanks @rainlance for your PR!

> Should I make a new pull request in order to remove changes in line 47?

You don't need to make a new pull request. You can amend your commit with `git commit --amend amath-1.8.5.recipe` and then "force push" it with `git push -f`.

BTW, I just noticed that your commit is in your master branch.
While this is not a problem at all, many users prefer to use [feature-branches](https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches).

https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches
This allows you to keep your master branch perfectly in sync with the project's master, and even easily refresh your master branch at any time, without having to touch your feature-branches.
That said, you are perfectly free to submit anything from your master branch. Using feature-branches is just a recommendation, and not using them does not have any impact.

(It's fine to continue with this PR. Actually, it is perfectly OK, and even (slightly) better than closing it and opening a new PR, but if you prefer to do that, it's also OK.)

If you wish I can "fix" your commit in this PR for you. (I'm used to updating PRs, so it's an easy task for me.).
BTW, it won't have any impact on the final result and you will still be the author of the commit.
Shall I do so?

Reply:
```diff
 TEST()
 {
+	LIBRARY_PATH="$sourceDir/src/lib:$sourceDir/src/cplex:$sourceDir/src/real:$sourceDir/src/clib${LIBRARY_PATH:+:$LIBRARY_PATH}" \
 	make test
}
```
BTW, amath builds fine on x86_64 (**without** the `--without-stdc++`) and passes most tests on this platform:
```
Passed: 541, failed: 1
```
The only test which fails is:
```
FAIL: [coth(1+2i)] expected [coth(1+2i) = 0.82132979749385+0.17138361290919i] but got [coth(1+2i) = 0.82132979749385+0.17138361290918i]
```
The difference is in the very last digit of the imaginary part:

Expected | 0.17138361290919
-- | --
Got | 0.17138361290918

Reply:
And thank you for the thoroughly review !